components/impls/core/src/main/javascript/components/NewMapComponent/amd/engines/google/MapEngineGoogle.js

```diff
@@ -716,59 +716,70 @@ define([
             var url;
             if (_.isArray(urlList)) {
               url = myself._selectUrl(
-                _.template(
-                  "${z}/${x}/${y}",
-                  {interpolate: /\$\{(.+?)\}/g}
-                )({x: coord.x, y: coord.y, z: zoom}),
-                urlList
+                  _.template(
+                      "${z}/${x}/${y}",
+                      {interpolate: /\$\{(.+?)\}/g}
+                  )({x: coord.x, y: coord.y, z: zoom}),
+                  urlList
               );
             } else {
               url = urlList;
             }
             return _.template(
-              url,
-              {interpolate: /\$\{(.+?)\}/g}
-            )({x: coord.x,y: coord.y, z: zoom});
+                url,
+                {interpolate: /\$\{(.+?)\}/g}
+            )({x: coord.x, y: coord.y, z: zoom});
           }
         }
       }, options));
     },
 
-    showPopup0: function(data, feature, popupHeight, popupWidth, contents, popupContentDiv, borderColor) {
+    showPopup0: function (data, feature, popupHeight, popupWidth, contents, popupContentDiv, borderColor) {
       if (popupContentDiv && popupContentDiv.length > 0) {
         contents = $("#" + popupContentDiv).html();
       }
 
       var popup = new OurMapOverlay(feature.getGeometry().get(), popupWidth, popupHeight, contents, popupContentDiv, this.map, borderColor);
       this._popups = this._popups || [];
-      _.each(this._popups, function(p) {
+      _.each(this._popups, function (p) {
         p.setMap(null);
       });
       this._popups.push(popup);
     },
 
-    showPopup: function(data, feature, popupHeight, popupWidth, contents, popupContentDiv, borderColor) {
+    showPopup: function (data, feature, popupHeight, popupWidth, contents, popupContentDiv, borderColor) {
```

Comment:
Ideally, you'd rename the `popupContentDiv` parameter to `popupContentId`.
Additionally, `contents` could be renamed to `popupContent`.
The naming, as it is, is quite confusing.